pom.xml

```diff
@@ -172,6 +172,12 @@
             <version>1.1.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>${junit5.version}</version>
```

Comment:
Where is this being defined?

Reply:
It is using the one defined in parent POM (dropwizard). Updated in this PR https://github.com/alphagov/pay-ledger/pull/822/files#diff-600376dffeb79835ede4a0b285078036R18